src/components/Modal.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Photo from './Photo.jsx';
+import Close from './ExitButton.jsx';
+
+
+function Modal({ heroPhoto, swipeToLast, swipeToNext, exitModal, isOpen}) {
+  return (
```

Comment:
maybe consider using a conditional render here and not show the previous button on the first picture and not show the next button on the last picture.

Reply:
Good idea! I was going to try a linked list but this may be a faster approach.